pages/staff.vue

```diff
@@ -0,0 +1,151 @@
+<template>
+	<main>
+		<navbar />
```

Comment:
Since you used navbar twice now, we should put it into a layout and utilize that. For pages without navbar we simply need to use a different layout https://nuxtjs.org/docs/2.x/directory-structure/layouts

Reply:
Layouts created.